<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicFilters</class>
 <widget class="QDialog" name="OBSBasicFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic.Filters</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="asyncWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="asyncLabel">
            <property name="text">
             <string>Basic.Filters.AsyncFilters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FocusList" name="asyncFilters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::TargetMoveAction</enum>
            </property>
            <property name="spacing">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="widget">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addAsyncFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>Add</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-plus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeAsyncFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>Remove</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-trash</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveAsyncFilterUp">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>MoveUp</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveAsyncFilterDown">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>MoveDown</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-down</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="asyncToolbarSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="separatorLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="effectWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Basic.Filters.EffectFilters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FocusList" name="effectFilters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::TargetMoveAction</enum>
            </property>
            <property name="spacing">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="widget_2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addEffectFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>Add</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-plus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeEffectFilter">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>Remove</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-trash</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveEffectFilterUp">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>MoveUp</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moveEffectFilterDown">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="accessibleName">
                <string>MoveDown</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="obs.qrc">
                 <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">btn-tool icon-down</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="effectToolbarSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="rightContainerLayout">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="rightLayout">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QFrame" name="previewFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="OBSQTDisplay" name="preview" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="propertiesFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="propertiesLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Close|QDialogButtonBox::RestoreDefaults</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRemoveFilter">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveUp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveDown</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionRenameFilter">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OBSQTDisplay</class>
   <extends>QWidget</extends>
   <header>qt-display.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusList</class>
   <extends>QListWidget</extends>
   <header>focus-list.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
